development/mimir-ingest-storage/config/grafana-agent.flow

```diff
@@ -36,6 +36,13 @@ prometheus.scrape "metrics_local_mimir_read_write_mode_mimir_write" {
                         container   = "mimir-write",
                         namespace   = "mimir-read-write-mode",
                 }],
+                                [{
+                                        __address__ = "mimir-write-zone-c-61:8080",
+                                        cluster     = "docker-compose",
+                                        container   = "mimir-write",
+                                        namespace   = "mimir-read-write-mode",
+                                        job = "mimir-write-zone-c",
+                                }],
```

Comment:
(nit) Seems that tabs and spaces are mixed in this config here, right? ;)

Reply:
i think it's only using spaces.. i asked goland to convert "to spaces" and it said it's already formatted. i fixed the indentation of this line in the meantime

Reply:
@dimitarvdimitrov have you forgotten to push the changes, maybe? The indentation is still off here ;)

Reply:
this is what i pushed https://github.com/grafana/mimir/pull/9455/commits/f5b6bcbdff6d5f1cb4f2c8207aecd753f61e7959
<img width="1227" alt="Screenshot 2024-10-10 at 11 19 41" src="https://github.com/user-attachments/assets/74bbb0b1-28ad-403c-8441-7bb31ac06970">

i think GH doesn't update the diff on the comment